client/my-sites/marketing/connections/mailchimp-settings.jsx

```diff
@@ -111,7 +113,7 @@ const MailchimpSettings = ( {
 				<Notice
 					status="is-warning"
 					text={ translate(
-						'Subscribers will not be added to Mailchimp for this site. Please select a list to sign them up for your Mailchimp content'
+						'Subscribers will not be added to Mailchimp for this site. Please select an audience to sign them up for your Mailchimp content'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.